### PR TITLE
Set up linting and change files according to linting rules

### PR DESCRIPTION
**Pull request by anthonytate**

### CODE REVIEW

**Review by MrM117 — COMMENTED** [review]
You can avoid the build errors by giving the elements ids (e.g., id="showPictureAndCaptionButton") and installing the event listeners in the js file (e.g. document.getElementById(...).addEventListener(...))

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -27,32 +27,34 @@ const MILKSHAKE_IMG_CAPTION = 'Milkshakes are my favorite food/drink. I had ' +
...
-    CAPTION_CONTAINER.innerHTML = IMG_CAPTION;
+  const picturesAndCaptions = [
+    ['images/Snow.jpg', SNOW_IMG_CAPTION],
+    ['images/Chicago.jpg', CHICAGO_IMG_CAPTION],
+    ['images/Baby.jpg', BABY_IMG_CAPTION],
+    ['images/Milkshake.JPG', MILKSHAKE_IMG_CAPTION],
+    ['images/Fish.PNG', FISH_IMG_CAPTION],
+  ];
+
+  const IMG_INDEX = Math.floor(Math.random() * 5);
```
Comment (MrM117) [inline portfolio/src/main/webapp/script.js]: These local variables, despite being const, would be better named as local variables (i.e. lowerCamelCase).  See https://google.github.io/styleguide/jsguide.html#naming-constant-names

Comment (anthonytate) [reply portfolio/src/main/webapp/script.js]: Done

**Review by MrM117 — APPROVED** [review]
LGTM

**Pull request merged**